test/app/smsController.js

```diff
@@ -0,0 +1,120 @@
+const assert = require('assert');
+const supertest = require('supertest');
+const should = require('should');
+
+const APP = require('../../app/app');
+
+const Emails = require('../../app/models/emails');
+const Capture = require('../../app/models/capture');
+const CommConns = require('../../app/models/commConns');
+const Communications = require('../../app/models/communications');
+const Conversations = require('../../app/models/conversations');
+const Messages = require('../../app/models/messages');
+
+const twilioAgent = supertest.agent(APP);
+const smsData = require('../testSmsData');
+
```

Comment:
So the takeaway here is add model level tests to pair with these controller level tests. Will do.